scripts/multisrc/lightnovelwp/sources.json

```diff
@@ -166,5 +166,14 @@
       "reverseChapters": true,
       "seriesPath": "/novels/"
     }
+  },
+  {
+    "id": "lightnovelbrasil",
+    "sourceSite": "https://lightnovelbrasil.com/",
+    "sourceName": "Light Novel Brasil",
+    "options": {
+      "lang": "Portuguese",
+      "reverseChapters": true
+    }
```

Comment:
this need versionIncrements

Reply:
It was a separate file before so it was version 1.0.0 before

Reply:
the old one didnt work, does this work?

Reply:
Yes

Reply:
oh, I forgot this template version is 1.0.4